backend/YouTubeV2.Application/Services/UserService.cs

```diff
@@ -186,7 +186,8 @@ public async Task<UserDto> GetDTOForUser(User user, bool getAllData = false, Can
             var imageUri = _blobImageService.GetProfilePicture(user.Id);
             var roleName = await GetRoleForUserAsync(user);
             decimal? accountBallance = getAllData ? user.AccountBalance : null;
-            int? subscriptionsCount = await _subscriptionService.GetSubscriptionCountAsync(user.Id, cancellationToken);
+            int? subscriptionsCount = await _userManager.IsInRoleAsync(user, Role.Creator) ?
```

Comment:
Im not so sure about this null as a subscription count. imo just return 0

Reply:
in spec this is nullable